src/flexmeasures_client/client.py

```diff
@@ -716,6 +747,19 @@ async def update_sensor(self, sensor_id: int, updates: dict) -> dict:
             )
         return updated_sensor
 
+    async def delete_sensor(self, sensor_id: int):
+        """Deletes a sensor and its data.
+
+        This function raises a ValueError when an unhandled status code is returned.
+        """
+        answer = input(f"Delete sensor {sensor_id} and all its data? [yN]")
+        if answer.lower() not in ["y", "yes"]:
+            print("Aborting ...")
+            return {}
```

Comment:
Why return an empty dict in case of aborting, and nothing in case of proceeding?